kusto/test/etoe/etoe_test.go

```diff
@@ -41,9 +40,9 @@ func (c *config) validate() error {
 	case c.Endpoint, c.Database, c.ClientID, c.ClientSecret, c.TenantID:
 		return fmt.Errorf("no field in the end to end test config.json file can be empty")
 	}
-	if len(c.Principals) == 0 {
-		return fmt.Errorf("config.json cannot have an empty Principals field")
-	}
+	//if len(c.Principals) == 0 {
```

Comment:
Did you mean to comment this out?  If so, why not remove it?

Reply:
hmm I was actually meaning to ask you if you think it's important. seems like a very edgy test that is covered in other tests.

I'll remove it and the test itself